dataflow_test.py

```diff
@@ -513,17 +513,16 @@ def test_pipes_must_end_in_a_sink():
 
 
 def test_count_filter_filters_correctly():
-    even_numbers = list(range(0, 20, 2))
-    odd_numbers  = list(range(1, 20, 2))
-    the_source   = odd_numbers + even_numbers
+    the_source  = list(range(21))
+    predicate   = lambda n: n % 2
 
-    odd      = df.count_filter(lambda n: n % 2)
+    odd      = df.count_filter(predicate)
     filtered = []; the_sink = df.sink(filtered.append)
 
     df.push(source = the_source,
             pipe   = df.pipe(odd.filter, the_sink))
 
-    assert filtered == odd_numbers
+    assert filtered == list(filter(predicate, the_source))
```

Comment:
Yup, I don't think we should make any secret or apology about the similarity between our `map`, `filter` and `reduce` and the standard ones. So I'd say it absolutely right to highlight these similarities in the tests.